keps/sig-node/5365-ImageVolume-with-image-digest/README.md

```diff
@@ -0,0 +1,978 @@
+    plane? No.
+  - Will enabling / disabling the feature require downtime or reprovisioning
+    of a node? Yes.
+
+###### Does enabling the feature change any default behavior?
+
+<!--
+Any change of default behavior may be surprising to users or break existing
+automations, so be extremely careful here.
+-->
+
+By default, the image ID should be added to the pod's status when the feature is enabled and an ImageVolume is used.
+This should not break any existing flows / behaviors.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+<!--
+Describe the consequences on existing workloads (e.g., if this is a runtime
+feature, can it break the existing applications?).
+
+Feature gates are typically disabled by setting the flag to `false` and
+restarting the component. No other changes should be necessary to disable the
+feature.
+
+NOTE: Also set `disable-supported` to `true` or `false` in `kep.yaml`.
+-->
+
+The feature gate can be turned off.
+Kubelet would be needed to be restarted for the image digest to be removed from the pod's status.
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+After re-enabling the feature, any new pod that would be created would include the image digest in its status.
+Vice-versa for disabling the feature.
+
+###### Are there any tests for feature enablement/disablement?
+
+<!--
+The e2e framework does not currently support enabling or disabling feature
+gates. However, unit tests in each component dealing with managing data, created
+with and without the feature, are necessary. At the very least, think about
+conversion tests if API types are being modified.
+
+Additionally, for features that are introducing a new API field, unit tests that
+are exercising the `switch` of feature gate itself (what happens if I disable a
+feature gate after having objects written with the new field) are also critical.
+You can take a look at one potential example of such test in:
+https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282
+-->
+
+### Rollout, Upgrade and Rollback Planning
+
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+<!--
+Try to be as paranoid as possible - e.g., what if some components will restart
+mid-rollout?
+
+Be sure to consider highly-available clusters, where, for example,
+feature flags will be enabled on some API servers and not others during the
+rollout. Similarly, consider large clusters and how enablement/disablement
+will rollout across nodes.
+-->
+
+The worst case scenario is that some pods would have the image digest in their status, while others would not.
+This is not expected to cause any issues or break existing behavior.
+
+###### What specific metrics should inform a rollback?
```

Comment:
seems like pod/status update failures would be an indication that something here may have broken and be a good indication.